src/resolvers.ts

```diff
@@ -79,6 +80,7 @@ export interface ResolverOptions<T> {
    * Lifetime setting.
    */
   lifetime?: LifetimeType
+  defaultArgs?: any
```

Comment:
Shouldn't this be a `Map`?

```suggestion
  /**
   * A map of dependencies and their default value in case
   * there is no registration in the container.
   */
  defaultArgs?: Map<string, any>
```

Reply:
It should! I started off with `any` for simplicity and forgot to change it.